ui-tests/cypress/support/pageObjects/administration/fileBrowser/CreateFolderPage.js

```diff
@@ -6,18 +6,33 @@ import AppPage from '../../app/AppPage';
 
 import UploadFilesPage    from './UploadFilesPage';
 import CreateTextFilePage from './CreateTextFilePage';
+import FilesListPage from './FilesListPage';
 
 export default class CreateFolderPage extends Content {
 
-  breadCrumbs      = `${htmlElements.ol}.breadcrumb`;
-  operationButtons = `${htmlElements.div}.btn-group`;
+  breadCrumbs        = `${htmlElements.ol}.breadcrumb`;
+  operationButtons   = `${htmlElements.div}.btn-group`;
+
+  createFolderForm   = `${htmlElements.form}.FileBrowserCreateFolder`;
+  folderNameInput    = `${htmlElements.input}[name='path']#path`;
+  nameInputHelpBlock = `${htmlElements.span}.help-block`;
+  cancelButton       = `${htmlElements.a}.FileBrowserCreateFolderForm__btn-cancel`;
+  saveButton         = `${htmlElements.button}.FileBrowserCreateFolderForm__btn-submit`;
 
   getFileBrowserBreadCrumbs() {
     return this.getContents()
                .children(htmlElements.div).eq(2)
                .find(this.breadCrumbs);
   }
 
+  clickFileBrowserBreadCrumbs(element) {
```

Comment:
Why it is needed to intercept the API call to click the breadcrumb?

Reply:
The intercept + wait is to help cypress wait for the page to update. The intercept has been moved to the beforeEach